### ISSUE TITLE
Page stuck at loading "Start creating views" 

### ISSUE THREAD

**Issue by fishfree** [opencast/annotation-tool#177]
F12, there is error: Uncaught TypeError: Cannot read property 'mediapackage' of undefined
Attach the screenshot: 
![annotationerror](https://user-images.githubusercontent.com/1741341/45696312-e87fa200-bb95-11e8-9824-5ac29e4d9ee7.jpg)

**Comment by JulianKniephoff** [comment]
This looks like Chrome on Windows, can you confirm that?

**Comment by fishfree** [comment]
@JulianKniephoff Yes, Chrome 68+ on Windows 10.

**Comment by fishfree** [comment]
@JulianKniephoff I also tried on Edge, Windows 10, the same problem. I don't think it's dependent on browsers or OS.

What's worse is a 404 error occured when I recompiled Opencast & Annotation tool and deployed the annotation tool, because an error occured when compiling Opencast. I tried several days with no luck. So it will be very kind of you to provide a compiled jar file of this annotation tool for Opencast 5.0. I compiled them in Ubuntu 16.04.
